Reqnroll/Formatters/ExecutionTracking/FeatureExecutionTracker.cs

```diff
@@ -54,9 +54,8 @@ public FeatureExecutionTracker(FeatureStartedEvent featureStartedEvent, string t
         StepDefinitionsByBinding = stepDefinitionIdsByMethod;
         IdGenerator = idGenerator;
         FeatureName = featureStartedEvent.FeatureContext.FeatureInfo.Title;
-        var featureHasCucumberMessages = featureStartedEvent.FeatureContext.FeatureInfo.FeatureCucumberMessages != null && featureStartedEvent.FeatureContext.FeatureInfo.FeatureCucumberMessages.GherkinDocument != null;
-        featureHasCucumberMessages = featureHasCucumberMessages && featureStartedEvent.FeatureContext.FeatureInfo.FeatureCucumberMessages.GherkinDocument() != null;
-        Enabled = featureHasCucumberMessages;
+        Enabled = featureStartedEvent.FeatureContext.FeatureInfo.FeatureCucumberMessages is not null &&
+                                         featureStartedEvent.FeatureContext.FeatureInfo.FeatureCucumberMessages.HasMessages;
```

Comment:
nitpick, the spacing looks strange.

And could we do this in one line?

`Enabled = featureStartedEvent.FeatureContext.FeatureInfo.FeatureCucumberMessages?.HasMessages ?? false`

Reply:
Fixed.